mappings/net/minecraft/level/Level.mapping

```diff
@@ -1,27 +1,41 @@
 CLASS fd net/minecraft/level/Level
 	FIELD B isClient Z
 	FIELD d players Ljava/util/List;
+	FIELD h unusedIncrement I
+	FIELD i rainGradientPrev F
+	FIELD j rainGradient F
+	FIELD k thunderGradientPrev F
+	FIELD l thunderGradient F
+	FIELD q difficulty I
 	FIELD r rand Ljava/util/Random;
 	FIELD v cache Lcl;
 	FIELD x properties Lei;
 	METHOD <init> (Lwt;Ljava/lang/String;Lxa;J)V
 		ARG 4 seed
 	METHOD A canSkipNight ()Z
-	METHOD a isPlayerInRange (DDDD)Lgs;
+	METHOD E initWeatherGradients ()V
+	METHOD F resetWeather ()V
+	METHOD a getClosestPlayer (DDDD)Lgs;
 	METHOD a playSound (DDDLjava/lang/String;FF)V
 	METHOD a setTileEntity (IIILow;)V
 	METHOD a setLevelTime (J)V
 	METHOD a setSpawnPosition (Lbr;)V
 	METHOD a getEntities (Ljava/lang/Class;Leq;)Ljava/util/List;
+	METHOD a getPlayerByName (Ljava/lang/String;)Lgs;
 	METHOD a addParticle (Ljava/lang/String;DDDDDD)V
-	METHOD a getClosestPlayer (Lsn;D)Lgs;
+	METHOD a getClosestPlayerTo (Lsn;D)Lgs;
```

Comment:
Why

Reply:
Thought it'd make it clearer that it uses the passed entity as the center for finding nearby players.

Reply:
Ok